task-launcher/src/tasks/shared/helpers/fetchAndParseCorpus.js

```diff
@@ -46,6 +49,7 @@ const transformCSV = (csvInput, numOfPracticeTrials, sequentialStimulus) => {
       task: row.task,
       // for testing, will be removed
       prompt: row.prompt,
+      origItemNum: row.orig_item_num,
```

Comment:
Can you add this property conditionally if roarDefaults is true.

```suggestion
 if (roarDefaults) {
      newRow.origItemNum = row.orig_item_num,
 }
```